### ISSUE TITLE
Intracytoplasmic myofiber reducing inclusion bodies

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7628]
**Preferred term label:**
Intracytoplasmic myofiber reducing inclusion bodies

**Synonyms**

**Definition (free text, please give PubMed ID)**
Intracytoplasmic inclusions in myofibers that reduce nitro-blue tetrazolium (NBT) and thus stain strongly with the menadione-NBT stain 

See Figure 1 of PMID:18274675

**Parent term (use hpo.jax.org/app)**
Muscle fiber cytoplasmatic inclusion bodies HP:0100303

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
300718 
PMID:18274675  2/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
also  PMID: 18952429   1/1

**Comment by pnrobinson** [comment]
added
